apps/console/src/public/deployment.config.json

```diff
@@ -608,6 +608,25 @@
                     ]
                 }
             },
+            "parentUserInvitation": {
```

Comment:
Add a `j2` mapping as well.

Reply:
And also update the `default.json` in `carbon-identity-framework`

Reply:
`J2 ` mapping updated and the `default.json` in `carbon-identity-framework` is updated with https://github.com/wso2/carbon-identity-framework/pull/5672